mev_inspect/schemas/swaps.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from mev_inspect.schemas.classified_traces import Protocol
+
+
+class Swap(BaseModel):
```

Comment:
What do you think of an additional property like `slippage_tolerance` that'd calculate it based on historical reserves (when the tx was included). Default is 0.5% on uniswap but I've a feeling it'd come in handy when we implement sandwich detection or just want to get a better idea.

Reply:
Is that included as a parameter of the swap usually? If so I'm definitely for it.

If it requires some additional lookups / calculation, I think it'd be better as an added attribute later to keep this object to the data that's derivable directly from the trace (similar argument for separating `profit_amount_eth` on `Arbitrage`, keep it purely un-opinionated at this level)

Reply:
One of the param's expresses it (ex: `AmountOutMin` in `swapExactTokensForETH`) but you'd still need to query historical pool reserves to get the exact %. As you said, it makes sense to keep this un-opinionated so we can do this pre-db insert.